pkg/minikube/extract/extract.go

```diff
@@ -173,6 +173,11 @@ func inspectFile(e *state) error {
 		return err
 	}
 
+	if strings.HasSuffix(e.filename, "err_map.go") {
```

Comment:
This seems brittle in the case of the filename changing. Could you make this a compile-time failure if the path changes?

Two ideas: Use the filename as a constant in err_map, or `reflect.PkgPath()`. 

Thoughts?

Reply:
I felt it was easier to explicitly check whether the file existed so I went ahead and did that.